### ISSUE TITLE
Cloning recursively with submodules

### ISSUE THREAD

**Issue by tilsche** [icfpcontest2020/dockerfiles#65]
Are repositories cloned recursively such that we can, for example, include header-only libraries in submodules?

**Comment by beevee** [comment]
Yes, we do clone them recursively, as far as I can tell. Can you create a test submission with submodules just to check if it builds correctly?

**Comment by tilsche** [comment]
Yes, it actually works, thanks.
